### PR TITLE
Upgrade TangoDatabase to Release-5.12

### LINKED ISSUE #41: Provide an DB upgrade script from 9.2.5 to 9.3.3
Hello,

this is juste that dbconfig-common does an incremental upgrade from on version to the other.
So I just need a script whcih does the migration from 9.2.5 to 9.3.3

DELETE FROM property_class WHERE class='DServer' AND count >= 11;
INSERT INTO property_class VALUES('DServer','AllowedAccessCmd',11,'EventConfirmSubscription',NOW(),NOW(),NULL);

Is it the only change  ?

_Originally posted by @picca in https://github.com/tango-controls/TangoSourceDistribution/pull/18#issuecomment-498024593_

### PR DESCRIPTION
**Pull request by mliszcz**
Continuation of tango-controls/TangoDatabase#18.
Finally fixes #41.

### CODE REVIEW

File: assets/README [hunk 1]
```diff
@@ -412,6 +412,28 @@ The "make install" command will create these database related scripts in
 done at configure time is successfull ("database schema create" reported
 as yes at the end of the configure script execution)
 
+
+7-2 - UPDATING FROM A TANGO 9.2.5 DATABASE
```
Comment (bourtemb) [inline assets/README]: Very strange section number (was already like that before)...
I would start with 7-1 and then change the numbers of the following 7-x sections (7-2, 7-3, etc...).

Comment (mliszcz) [reply assets/README]: Thanks! Fixed as proposed.

File: assets/README [hunk 2]
```diff
@@ -412,6 +412,28 @@ The "make install" command will create these database related scripts in
 done at configure time is successfull ("database schema create" reported
 as yes at the end of the configure script execution)
 
+
+7-1 - UPDATING FROM A TANGO 9.2.5 DATABASE
+--------------------------------------
+
+Tango 9.3.3 includes an update of the database:
+(1) new DServer command: EventConfirmSubscription,
```
Comment (bourtemb) [inline assets/README]: new DServer command: EventConfirmSubscription,  => new DServer always allowed command: EventConfirmSubscription,

Comment (mliszcz) [reply assets/README]: Done

**Review by bourtemb — APPROVED** [review]
LGTM! Thanks @mliszcz!

**Pull request merged**